toxiproxy.go

```diff
@@ -12,7 +12,7 @@ var host string
 var port string
 
 func init() {
-	flag.StringVar(&host, "host", "localhost", "Host for toxiproxy's API to listen on")
+	flag.StringVar(&host, "host", "0.0.0.0", "Host for toxiproxy's API to listen on")
```

Comment:
This is so it binds to all interfaces in Docker which makes sure that if you're using native networking it binds on the `docker0` interface so you can access it from other containers.